backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/CreateNewWorkService.java

```diff
@@ -0,0 +1,171 @@
+package de.bund.digitalservice.ris.norms.application.service;
+
+import de.bund.digitalservice.ris.norms.domain.entity.*;
+import de.bund.digitalservice.ris.norms.domain.entity.eli.DokumentManifestationEli;
+import de.bund.digitalservice.ris.norms.domain.entity.eli.NormExpressionEli;
+import de.bund.digitalservice.ris.norms.domain.entity.metadata.rahmen.RahmenMetadata;
+import de.bund.digitalservice.ris.norms.utils.NodeCreator;
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.stereotype.Service;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+/**
+ * Service for creating new expressions and manifestations of a norm.
+ */
+@Service
+public class CreateNewWorkService {
+
+  /**
+   * Creates a completely new work out of an embedded norm, the Rechtsetzungsdokument of the Verkündung and the generated expression eli for the new norm computed by the zielnorm-reference
+   * The Rechtsetzungsdokument of the new work is created out of the newly created Regelungstext. The redok metadata are copied from the Rechtsetzungsdokument of the Verkündung.
+   * @param embeddedNorm taking out of the Verkündung (not loaded from the DB)
+   * @param rechtsetzungsdokumentVerkuendung needed for the redok metadata of the new Rechtsetzungsdokument of the new work
+   * @param expressionEli the expression eli to be used for the new work
+   * @return the new work
+   */
+  public Norm createNewWork(
+    Norm embeddedNorm,
+    Rechtsetzungsdokument rechtsetzungsdokumentVerkuendung,
+    NormExpressionEli expressionEli
+  ) {
+    var newNorm = new Norm(embeddedNorm);
+    newNorm
+      .getDokumente()
+      .forEach(dokument -> {
+        final DokumentManifestationEli dokumentManifestationEli = dokument.getManifestationEli();
+        final String newNaturalIdentifier = expressionEli.getNaturalIdentifier();
+        final LocalDate newPointIntTime = expressionEli.getPointInTime();
+        dokumentManifestationEli.setNaturalIdentifier(newNaturalIdentifier);
+        dokumentManifestationEli.setPointInTime(newPointIntTime);
+        dokumentManifestationEli.setPointInTimeManifestation(Norm.WORKING_COPY_DATE);
+        final String newSubtype = dokumentManifestationEli
+          .getSubtype()
+          .replaceAll("-(\\d+)$", "-1");
```

Comment:
i think this doesn't work for other dokumente than regelungstexte? Eg. a `anlage-regelungstext-3` would now be called `anlage-regelungstext-1`

Reply:
Handled in [this](https://github.com/digitalservicebund/ris-norms/pull/1634) other PR.